### DISCUSSION TITLE
我可以帮忙做一些图标

### DISCUSSION THREAD

**Discussion by KyleBing** [imfuxiao/Hamster#217] [Ideas]
目前看到项目中的菜单图标不是很完美，如果新开发进程中有 icon 需求，我可以帮忙做一些供参考。

方便的话，可以提供一些界面截图，以及图标大体位置。

<img width="1302" alt="Screenshot 2023-05-20 at 11 40 11" src="https://github.com/imfuxiao/Hamster/assets/12215982/be12ed9b-fca7-4316-8c5d-40ea470b1765">
<img width="2032" alt="Screenshot 2023-05-20 at 11 40 59" src="https://github.com/imfuxiao/Hamster/assets/12215982/94963c35-8d04-4cfc-833b-2806ef0220b9">

**Comment by imfuxiao** [comment]
赞，非常感谢，目前icon是用内置的，有些可能不符合内容。后面有需要，我找您商量。

**Reply by KyleBing** [reply]
OK  https://kylebing.cn/
